catkit/hub/ase_tools/__init__.py

```diff
@@ -289,6 +286,21 @@ def get_layers(atoms):
     return layer_i
 
 
+def get_layers(atoms):
+    # WARNING: this function is defined twice
+    # with different parameter choices
+    tolerance = 0.01
+    d = atoms.positions[:, 2]
+    keys = np.argsort(d)
+    ikeys = np.argsort(keys)
+    mask = np.concatenate(([True], np.diff(d[keys]) > tolerance))
+    layer_i = np.cumsum(mask)[ikeys]
+
+    if layer_i.min() == 1:
+        layer_i -= 1
+    return layer_i
+
+
```

Comment:
That sounds like a good plan. The less code we have to maintain the better. If there are any other functions which aren't being maintained by you or max we should trim those out too. This will also help our testing score :).

I can show you how to tags layers with catgen (or ase also has a function for this) whenever you like.